into/backends/sql_csv.py

```diff
@@ -6,13 +6,12 @@
 
 import sqlalchemy
 
-from multipledispatch import MDNotImplementedError
-
 from ..regex import RegexDispatcher
 from ..append import append
 from .csv import CSV, infer_header
 from ..temp import Temp
 from ..into import into
+from .aws import S3
```

Comment:
It doesn't need to be because it's only when you instantiate it will it raise, like this:

```python
S3(CSV)('s3://mybucket/mykey.csv')
```

`S3(CSV)` won't raise